classes/patterns/class-action.php

```diff
@@ -0,0 +1,81 @@
+<?php
+/**
+ * Action pattern class.
+ *
+ * @package P4GBKS
+ * @since 0.1
+ */
+
+namespace P4GBKS\Patterns;
+
+use P4GBKS\Patterns\Templates\Covers;
+
+/**
+ * Class Action.
+ *
+ * @package P4GBKS\Patterns
+ */
+class Action extends Block_Pattern {
+
+	/**
+	 * Returns the pattern name.
+	 */
+	public static function get_name(): string {
+		return 'p4/action-pattern-layout';
+	}
+
+	/**
+	 * Returns the pattern config.
+	 *
+	 * @param array $params Optional array of parameters for the config.
+	 */
+	public static function get_config( $params = [] ): array {
+		$media_link = esc_url( get_template_directory_uri() ) . '/images/placeholders/placeholder-546x415.jpg';
+
+		return [
+			'title'      => __( 'Action', 'planet4-blocks-backend' ),
+			'categories' => [ 'layouts' ],
+			'content'    => '
+				<!-- wp:group {"className":"block"} -->
+					<div class="wp-block-group">
+						<!-- wp:media-text {"mediaLink":"' . $media_link . '","mediaType":"image"} -->
+							<div class="wp-block-media-text is-stacked-on-mobile">
+								<figure class="wp-block-media-text__media">
+									<img src="' . $media_link . '" alt="' . __( 'Default image', 'planet4-blocks-backend' ) . '"/>
+								</figure>
+								<div class="wp-block-media-text__content">
+									<!-- wp:gravityforms/form /-->
+								</div>
+							</div>
+						<!-- /wp:media-text -->
+						<!-- wp:group {"backgroundColor":"grey-05","align":"full","className":"is-style-large-padding"} -->
+							<div class="wp-block-group alignfull has-grey-05-background-color has-background is-style-large-padding">
+								<!-- wp:group {"className":"container"} -->
+									<div class="wp-block-group container">
+										' . SideImageWithTextAndCta::get_config(
+										[
+											'media_position'    => 'right',
+											'title_placeholder' => __( 'The problem', 'planet4-blocks' ),
+										]
+									)['content'] . '
+									</div>
+								<!-- /wp:group -->
+							</div>
+						<!-- /wp:group -->
+						' . Covers::get_content( [ 'cover_type' => 'take-action' ] ) . '
+						<!-- wp:separator {"backgroundColor":"grey-20","className":"is-style-wide"} -->
```

Comment:
The separator ends up different in the editor (it doesn't have the backgorund color UI selected and gets an opacity property)
```
<!-- wp:separator {"opacity":"css","className":"has-text-color has-grey-20-color has-grey-20-background-color has-background is-style-wide"} -->
<hr class="wp-block-separator has-css-opacity has-text-color has-grey-20-color has-grey-20-background-color has-background is-style-wide"/>
<!-- /wp:separator -->
```
I tested repasting it after setting the bg color, maybe the alpha-channel class fixes the thing ? seems to work :shrug: 
```
<!-- wp:separator {"backgroundColor":"grey-20","className":"has-text-color has-grey-20-color has-grey-20-background-color has-background is-style-wide"} -->
<hr class="wp-block-separator has-text-color has-grey-20-color has-alpha-channel-opacity has-grey-20-background-color has-background is-style-wide"/>
<!-- /wp:separator -->
```

Reply:
Good catch! Your fix seems to work indeed, but since the text color is also defined I see this warning in the sidebar:
<img width="278" alt="Screenshot 2022-08-04 at 10 55 16" src="https://user-images.githubusercontent.com/6949075/182806604-fba93427-e05f-4d02-811e-7bf0ed0e9274.png">
But if I remove the text color from the attributes, then the background color gets unset too for some reason 🤷‍♀️ I guess it's not a big deal, maybe it's a core block issue 🤔

Reply:
I still prefer your version since at least the background color is properly selected in the sidebar!